docs/getting-started/dev-quick-start/index.md

```diff
@@ -0,0 +1,342 @@
+          "sling:resourceType": "wcm/dialogs/components/include",
+          "sling:orderBefore": "overline",
+          "path": "/libs/howlite/components/common/headingsize",
+          "include": {
+            "sling:resourceSuperType": "/libs/howlite/components/common/headingsize",
+            "label": "Overline size",
+            "name": "overlineSize",
+            "description": "Changes font size",
+            "s": {
+              "selected": true
+            },
+            "m": {
+              "selected": false
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+The above definition specifies the new `overlineSize` field. It is placed before `overline` field and uses heading size definition from Howlite, but with _small_ size selected by default.
+
+### Install changes 
+Run the following command to install the changes on your local environment.
+```shell
+mvn -f luna/core/pom.xml clean install -P autoInstallBundle
+```
+
+## Part D: Functional tests
+
+### Running functional tests
+
+We prepared sample functional tests using [Cypress](https://www.cypress.io/). They are executed during maven build. You can run them using `npm` on your local environment too. However, you have to add the test content before. We prepared two functional tests for the _Luna Title_ component. Use the following script to install them.
+```shell
+mvn -f tests/content/pom.xml clean install -P autoInstallPackage
+```
+
+Run them using the following command.
+```shell
+npm run-script test --prefix tests/end-to-end
+```
+
+If you execute them, they detect your changes for the _Luna Title_ and fail. You should get the following results.
+```
+Running:  lunatitle.cy.ts                                                                 (1 of 1)
+
+
+Luna Title component
+  1) renders correctly in preview mode
+  2) renders correctly in edit mode
+
+
+0 passing (8s)
+2 failing
+
+1) Luna Title component
+     renders correctly in preview mode:
+
+    AssertionError: expected '<h6.hl-title__heading.hl-title__heading--size-5>' to have CSS property 'font-size' with the value '20px', but the value was '25.008px'
+    + expected - actual
+
+    -'25.008px'
+    +'20px'
+    
+    ...
+
+2) Luna Title component
+     renders correctly in edit mode:
+
+    Timed out retrying after 4000ms
+    + expected - actual
+
+    { 'sling:resourceType': 'luna/components/lunatitle',
+       title: 'New heading',
+       showSubtitle: 'true',
+    -  overlineSize: 'hl-title__heading--size-5',
+       subtitle: 'New overline text',
+       'jcr:primaryType': 'nt:unstructured',
+       headingLevel: 'h1',
+       headingSize: 'hl-title__heading--size-2' }
+```
+
+Firstly, you updated the default font size of the overline to ensure consistency with the design. The first test detected the change and failed as a result. Secondly, you added a new property to the dialog for the component. It was recognised as well.
+
+
+### Update functional tests
```

Comment:
I'm missing here a couple of reasons why we do have those tests. My impression now is:
- there are tests and they failed
- you must update them, because they fail.

How about pointing a couple of reasons why we do have those tests:
- we introduced tests to automatically catch any regression
- thanks to the tests we have more confidence when introducing changes
- we believe that this approach enables us to spend less time on manual testing and regression fixes and more time on developing new features

Reply:
@malaskowski thanks. I update it.